src/applications/vaos/services/slot/transformers.js

```diff
@@ -0,0 +1,40 @@
+import moment from '../../utils/moment-tz';
+import { getTimezoneBySystemId } from '../../utils/timezone';
+import { FREE_BUSY_TYPES } from '../../utils/constants';
+
+/**
+ *
+ * The slot datetimes we get back from VistA include
+ * an offset of +00:00 that isn't actually accurate. The times returned are
+ * already in the time zone of the facility. In order to prevent
+ * moment from using this offset, we'll remove it initially, apply the correct
+ * timezone and then convert to UTC which is what FHIR uses
```

Comment:
I'm not positive we'll get UTC back from this endpoint from VSP. Might be worth creating a ticket to keep track of and verify later.

Reply:
Good call.  Created new ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/8421 to keep track of this